### PR TITLE
Update ad-sso.md

### PR DESCRIPTION
**Pull request by pradipd**
Fixing yaml in example 2

### CODE REVIEW

**Comment by sethmanheim** [comment]
Looks good -- #isgn-off

File: AKS-Hybrid/ad-sso.md [hunk 1]
```diff
@@ -180,34 +180,35 @@ apiVersion: rbac.authorization.k8s.io/v1
...
+rules:
+- apiGroups: ["", "extensions", "apps"]
+  resources: ["*"]
+  verbs: ["*"]
+- apiGroups: ["batch"]
+  resources:
+  - jobs
+  - cronjobs
+  verbs: ["*"]
+apiVersion: rbac.authorization.k8s.io/v1
```
Comment (HariPrasadV) [inline AKS-Hybrid/ad-sso.md:198]: Should this be below line 200, for the `kind: RoleBinding`

Comment (pradipd) [reply AKS-Hybrid/ad-sso.md:198]: yes. it is still not right.

**Comment by sethmanheim** [comment]
Just ran it through a YAML linter, and it validated fine, so - #sign-off

**Pull request merged**